chili-epfl/FROG
# [Storybook] Initial work of sessions view on the homepage

This PR lays down an initial storybook component for the session view on the homepage. It also moves the OverflowPanel component to `/imports/ui/components/OverflowPanel` and replaces the `Settings` icon used with the `MoreVert` icon. 

**Testing instructions:**
- Run `yarn start storybook`, navigate to the BaseLayout story. Have a look at the design. Purely visual. Components take dummy data as props and render it.
- Functionality of FROG should be unchanged
